packages/release-mastermind/types/index.d.ts

```diff
@@ -3,0 +1,265 @@
+import {
+  Condition,
+  IssueCondition,
+  PRCondition,
+  ProjectCondition,
+  ScheduleCondition
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
  ScheduleCondition,
```